pkg/collector/collector.go

```diff
@@ -1584,6 +1587,11 @@ func (c *Collector) registerWithSamplers(ctx context.Context) {
 
 				readyPct := (nodesWithResults.Len() * 100 / nodesWithSamplers.Len())
 				if nodesWithSamplers.Len() > 0 && readyPct > c.UtilizationServer.MinResultPctBeforeReady {
+					cyclesCount++
+					if cyclesCount > c.UtilizationServer.WaitSamplerRegistrationsBeforeServe {
+						c.UtilizationServer.ServeResults.Store(true)
```

Comment:
right, I am not sure when we would want to do this.  This is mostly to control the rollout and deletion of pods so that we are fully initialized with node data before sending it to prometheus.